lib/jekyll/configuration.rb

```diff
@@ -382,5 +383,23 @@ def check_maruku(config)
           "`_config.yml` file."
       end
     end
+
+    # Private: Checks if the `plugins` config is a String
+    #
+    # config - the config hash
+    #
+    # Raises a Jekyll::Errors::InvalidConfigurationError if the config `plugins`
+    # is a string
+    private
+    def check_plugins(config)
+      if config.key?("plugins") && config["plugins"].is_a?(String)
+        Jekyll.logger.error "Configuration Error:", "You specified the" \
+          " `plugins` config in your configuration file as a String, please" \
+          " use an Array instead. If you wanted to set the directory of your" \
+          " plugins, use the config key `plugins_dir` instead."
+        raise Jekyll::Errors::InvalidConfigurationError,
+          "plugins: #{config["plugins"].inspect}"
```

Comment:
Let's update this to `'plugins' should not be a string, but was: #{config["plugins"].inspect}. Use 'plugins_dir' instead.`